src/data/moves/move-attrs/telekinesis-attr.ts

```diff
@@ -0,0 +1,31 @@
+import { BattlerTagType } from "#enums/battler-tag-type";
+import type { EnemyPokemon } from "#field/enemy-pokemon";
+import type { Pokemon } from "#field/pokemon";
+import { AddBattlerTagAttr } from "#moves/add-battler-tag-attr";
+import type { Move } from "#moves/move";
+
+/**
+ * Attribute to apply {@link https://bulbapedia.bulbagarden.net/wiki/Telekinesis_(move) | Telikinesis'}
+ * effect. Attacks against the target bypass accuracy checks for 3 turns.
+ * @extends AddBattlerTagAttr
+ */
+export class TelekinesisAttr extends AddBattlerTagAttr {
+  constructor() {
+    super(BattlerTagType.TELEKINESIS, false, { failOnOverlap: true });
```

Comment:
```suggestion
    super(BattlerTagType.TELEKINESIS, false, { failOnOverlap: true, turnCountMin: 3 });
```
there was this before in init moves, is it not needed?

Reply:
`TelekinesisTag` always sets its turn count to 3, so it's unnecessary to define the turn count here.